fsharp-backend/tests/Tests/Serialization.Tests.fs

```diff
@@ -892,16 +922,17 @@ module GenericSerializersTests =
               [ { tlid = testTLID
                   name = "dbname"
                   nameID = 7UL
-                  pos = testPos
+                  pos = CT2Program.Position.toCT Values.ProgramTypes.testPos
                   cols =
                     [ { name = Some("colname")
                         nameID = 8UL
-                        typ = Some(PT.TInt)
+                        typ = Some(CT2Program.DType.toCT PT.TInt)
                         typeID = 9UL } ]
                   version = 1 } ]
-            userFns = testUserFunctions
-            userTypes = testUserTypes
-            packageFns = [ testPackageFn ]
+            userFns =
+              List.map CT2Program.UserFunction.toCT ProgramTypes.testUserFunctions
```

Comment:
It seems the tests here are sometimes for ClientTypes and sometimes for converted types. It feels like there should be tests for roundtrips from real types, right? Maybe also roundtrips from ClientTypes?

Reply:
done!